db/migrate/20200212122000_add_service_parameters_raw.rb

```diff
@@ -0,0 +1,7 @@
+class AddServiceParametersRaw < ActiveRecord::Migration[5.1]
+  def change
+    add_column :order_items, :service_parameters_raw, :jsonb
+
+    OrderItem.update_all("service_parameters_raw=service_parameters")
```

Comment:
Added this to duplicate `service_parameters` to `service_parameters_raw` as they are (i.e. no masking) as per the comment above https://github.com/RedHatInsights/catalog-api/pull/587/files#r381013823.

Verified the `update_all` call works here on the `change` method for both up and down migrations.